xen/drivers/passthrough/arm/Makefile

```diff
@@ -1,2 +1,6 @@
 obj-y += iommu.o
 obj-$(CONFIG_ARM_SMMU) += smmu.o
+obj-$(CONFIG_RCAR_IPMMU) += ipmmu-vmsa.o
+obj-y += io-pgtable.o
+obj-y += io-pgtable-arm.o
+obj-y += ipmmu-vmsa-plat.o
```

Comment:
Why other files are not under CONFIG_RCAR_IPMMU?

Reply:
Because I've missed it.